README.md

```diff
@@ -46,28 +46,74 @@ For example:
-    -PwithSslProtocols    : Specifies enabled SSL protocols. See Geode property `ssl-protocols`
-    -PwithSslCiphers      : Specifies enabled SSL chipher suites. See Geode property `ssl-ciphers`
-    -PwithSecurityManager : Flag to start Geode with the example implementation of SecurityManager
-    -PwithSniProxy        : Use SNI proxy topology.
-    -PwithSniProxyImage   : Provide an alternative Docker image coordinate for SNI proxy.
-    -PwithRouter          : Use router with SNI proxy topology.
-    -PwithRouterImage     : Provide an alternative Docker image coordinate for router.
-    -PwithGc              : Select which GC to use. Valid values CMS (default), G1, Z.
-    -PwithHeap            : Specify how large a heap the benchmark VMs should use, default "8g". Accepts any `-Xmx` value, like "32g".
-    -PwithThreads         : Specify how many threads to use when executing the benchmark. Default varies by benchmark.
-    -PwithWarmup          : Specify how long to warm up the benchmark in seconds. Default is 60 seconds.
-    -PwithDuration        : Specify how long to measure the benchmark in seconds. Default is 300 seconds.
-    --tests               : Specific benchmarks to run (--tests=PartitionedPutBenchmark)
-    -d                    : Debug
-    -i                    : Info
-```    
+### Options
+The benchmarks can take configuration options. Some using Gradle's `-P` flag and other, which adjust
+benchmark behavior, via Java system properties using `-D`.
+
+| Option                | Description |
+| --------------------- | ----------- |
+| `-Phosts`               | Hosts used by benchmarks on the order of client,locator,server,server (-Phosts=localhost,localhost,localhost,localhost) |
+| `-PoutputDir`           | Results output directory (-PoutputDir=/tmp/results) |
+| `-PtestJVM`             | Path to an alternative JVM for running the client, locator, and servers. If not specified JAVA_HOME will be used. Note all compilation tasks will still use JAVA_HOME. |
+| `-Pbenchmark.X`         | Where X is a benchmark configuration, defined below. |
+| `--tests`               | Specific benchmarks to run (--tests=PartitionedPutBenchmark) |
+| `-d`                    | Debug |
+| `-i`                    | Info |
+
+#### Benchmark Configuration
+##### Common
+These options may apply to all benchmarks.
+
+| Option                | Description |
+| --------------------- | ----------- |
+| withGc                | Select which GC to use. Valid values CMS (default), G1, Z. |
+| withHeap              | Specify how large a heap the benchmark VMs should use, default "8g". Accepts any `-Xmx` value, like "32g". |
+| withThreads           | Specify how many threads to use when executing the benchmark. Default varies by benchmark. |
+| withWarmup            | Specify how long to warm up the benchmark in seconds. Default is 60 seconds. |
+| withWarmup            | Specify how long to run the benchmark in seconds. Default is 300 seconds. |
```

Comment:
There are two "withWarmup" options listed here. I think the second one should be "withDuration".